src/powershellScripts.ts

```diff
@@ -1,5 +1,9 @@
 import * as os from "os";
 
+export function getPesterVersion() {
+    return `(Get-InstalledModule Pester).Version`;
```

Comment:
What happens if Pester is not found?

Reply:
See https://github.com/TylerLeonhardt/vscode-pester-test-adapter/pull/50#discussion_r644223199